package.json

```
@@ -44,15 +44,15 @@
         "@react-native-community/datetimepicker": "^5.1.0",
         "@react-native-community/masked-view": "^0.1.11",
         "@react-native-community/netinfo": "^8.1.0",
-        "@react-native-picker/picker": "^2.3.1",
+        "@react-native-picker/picker": "^2.4.1",

```

Just to be in pair with [react-components](https://github.com/ripe-tech/ripe-components-react-native/blob/6c22a438678617c64f8778d9d28b2f5a5e60e21d/package.json#L70) and [robin](https://github.com/ripe-tech/ripe-robin-revamp/blob/6360f1bdbce94d84e41d0ce48c9b4bb5346b2591/package.json#L44):
```suggestion
        "@react-native-picker/picker": "^2.4.4",

```

---

nice catch